### PR TITLE
Fix deprecated UIManager usage when accessing component names

### PR DESCRIPTION
**Pull request by declanelcocks**
### Does any other open PR do the same thing?

https://github.com/react-native-community/react-native-maps/pull/2671/files

### What issue is this PR fixing?

As mentioned in https://github.com/react-native-community/react-native-maps/issues/2620, there are some YellowBox warnings with deprecated `UIManager` calls being used. The PR above fixed some of the warnings, but a warning still pops up regarding `AIRMapLite` and `AIRMap`.

### How did you test this PR?

- Updated iOS application that uses `react-native-maps` to `react-native@0.58.0`
- Open screen with any `MapView` component
- Check that no warnings regarding deprecated `UIManager` usage exists

### CODE REVIEW

File: lib/components/MapView.js [hunk 1]
```diff
@@ -915,7 +937,7 @@ if (Platform.OS === 'android') {
 }
 const getAirMapComponent = provider => airMaps[provider || 'default'];
 
-const AIRMapLite = NativeModules.UIManager.AIRMapLite &&
+const AIRMapLite = NativeModules.UIManager.getViewManagerConfig('AIRMapLite') &&
```
Comment (vovkasm) [inline lib/components/MapView.js]: This probably will crash on RN <0.58 because UIManager does not have property getViewManagerConfig. Right?

Comment (declanelcocks) [reply lib/components/MapView.js]: @vovkasm Good point, I just pushed a commit to add a quick check for `UIManager.getViewManagerConfig` as with the other changes.

**Comment by rborn** [comment]
@vovkasm please let us know if the latest change it's ok and we merge 🎉

**Comment by vovkasm** [comment]
@rborn technically it looks correct. But I would recommend refactor the code to have logic in one place and not to spread hacks through source files.

**Comment by rborn** [comment]
@vovkasm @declanelcocks can you two please coordinate, maybe we get something better out of this 🤗

**Comment by vovkasm** [comment]
Ahm... Unfortunately I'm currently almost has no time to work on code.
@declanelcocks  Can you remove duplication of check logic from project, perhaps make one function to get view config from UIManager? If not, I think PR can be commited as is, because working code with duplication anyway better than no code :-)

**Comment by declanelcocks** [comment]
@vovkasm I can help to refactor it into a helper function, but probably won't get around to it until this weekend or early next week. I'd suggest the same as you and to merge this PR in, then make a new PR to refactor the `UIManager` logic as you mentioned.

**Comment by forki** [comment]
can you please merge as is and refactor later? this one is really important and fixes the map for RN 0.59.x

**Comment by rborn** [comment]
@declanelcocks @vovkasm I'll merge this one, please ping me when you refactor the code 🤗

**Comment by rborn** [comment]
@christopherdro maybe we should make a release so we have RN 0.59 working form npm?

**Comment by forki** [comment]
I can confirm master branch now works for me on android

**Comment by kumaresan-cgvak** [comment]
When can we expect a new release with the fix for RN 0.59?

**Comment by christopherdro** [comment]
We need to put together the release notes. 
I can try to do this weekend unless someone else wants to volunteer.

**Comment by rborn** [comment]
@forki @kumaresan-cgvak any of you fancy a changelog PR fort he release ?

this is the last published commit https://github.com/react-native-community/react-native-maps/commit/51ff72393bd81cc1df4bef2fa48fa1da0779ab31

and this is an example of a previous changelog: https://github.com/react-native-community/react-native-maps/pull/2666/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e

Thanks!

**Comment by LuiisFernando** [comment]
Sorry @rborn but I'm getting the same error running these versions

"react": "16.6.3",
"react-native": "0.58.6"
"react-native-maps": "^0.23.0",

If I install version 0.59 still getting runtime error on run react-native run-android
Any update to solve the problem in version 0.58.6 ?

**Comment by LuiisFernando** [comment]
How can I install master ?
using the command npm install react-native-maps --save it install version 0.23.0
And If am I correct the fix of this bug is a new version released march 16, right ?
How could I install this version to solve to yellow warning ??

Thanks!

**Pull request merged**